ccp/index_reader.py

```diff
@@ -648,7 +661,7 @@ def batch_process_projects(self,
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 9:
+    if len(sys.argv) != 11:
```

Comment:
have added the card for this to convert to argument processor already, will be part of separate PR.

Reply:
This is needed. Why we even went argv route here is beyond me

Reply:
It started with few arguments, and kept growing. Again, if we start using the Jenkins REST APIs in index_reader, the number of arguments to be passed would decrease as we can grab that via the APIs itself. Even if they are less, arg processor is needed.